README.md

```diff
@@ -1,19 +1,27 @@
 # FFC Payment Batch Processor
 
-FFC service to process Siti Agri batch files and send individual transactions as messages.
+## Description
 
-## Prerequisites
+FFC microservice acting as an adapter Siti Agri to process batch files and send individual transactions as messages to an Azure Service Bus topic.
+Receives batch payment files, validates payment requests and then publishes the validated requests.
```

Comment:
This feels a bit tacted on -- is a more specific version of the description above it...

Do we need it?